test/evergreen.yml

```diff
@@ -4167,10 +4167,10 @@ buildvariants:
     - name: verify-datafile-big-endian
     - name: verify-datafile-from-little-endian
 
-- name: ubuntu1804-ppc
-  display_name: "~ Ubuntu 18.04 PPC"
+- name: rhel-ppc
+  display_name: "~ RHEL8.1 PPC"
```

Comment:
Just "RHEL8 PPC" as rhel81 could be an alias name that the real OS being some other 8.x version.